css_tasks/task11/scss/_footer.scss

```diff
@@ -0,0 +1,220 @@
+@import "variables";
+@import "mixins";
+
+.footer {
+  background: $decorationBackground;
+}
+.footer__container {
+  position: relative;
+  display: flex;
+  padding-top: $standardTopPadding;
+  padding-bottom: 4.3125rem;
+}
+.footer__logo {
+  font-family: $secondaryFont;
+  font-size: calc(12px + (46 - 12) * ((100vw - 300px) / (1920 - 300)));
+  font-weight: bold;
+  text-decoration: none;
+  color: #ccc;
+}
+.footer__element {
+  flex-basis: 33%;
+  margin-right: 2rem;
+  p:first-child {
+    padding-top: 1.5rem;
+    margin-bottom: 2rem;
+    @include titleText;
+    font-weight: bold;
+    text-align: left;
+  }
+}
+.footer__element:first-child {
+  flex-basis: 40%;
+  p {
+    padding: 1.25rem 0;
+    @include normalText;
+    text-align: left;
+  }
+  p:nth-child(3) {
```

Comment:
classes?

Reply:
Yes sir. Just looking at this footer section a day later, I really see how big pile of shit I delivered :)